.travis.yml

```diff
@@ -0,0 +1,14 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+language: objective-c
```

Comment:
Travis does not provide explicit support for Swift - it works to just use `objective-c`.

"[Even though your project is in Swift, Travis only uses the objective-c value to know to build with Xcode command line tools.](https://www.raywenderlich.com/109418/travis-ci-tutorial)"